lib/omnibus/config.rb

```diff
@@ -400,14 +413,22 @@ def reset!
     #
     # @return [String]
     default(:publish_s3_access_key) do
```

Comment:
Am I missing a change in `S3Publisher.s3_configuration`??? This works as is -- but now I am questioning why the CLI seems to allow me to publish anyway via `S3Cache.populate`.